.github/workflows/cmake.yml

```diff
@@ -123,25 +141,51 @@ jobs:
         cat <<"EOF"
         ${{ toJSON(github.event) }}
         EOF
-    - name: Generate IRC message
-      if: matrix.should-notify-irc == true && !cancelled()
+    - name: Generate IRC message success
+      if: ${{ !cancelled() && needs.build_and_test_serenity.result && needs.build_and_test_lagom.result}}
+      run: |
+        ${{ github.workspace }}/Meta/notify_irc.py <<"EOF"
+        ["${{ github.actor }}", ${{ github.run_id }}, "success",
```

Comment:
Suggestion: Replace `"success"` by `["${{needs.build_and_test_serenity.result}}", "${{needs.build_and_test_lagom.result}}"]`.

And now that I think about it: How does this interact with the build matrix? After all, there are three different jobs all named `build_and_test_lagom`. Maybe add a `${{toJSON(needs)}}` above in "Dump event info", just to make sure no information is lost.

Also, my gut feeling says that this job needs to be put below the jobs it references.

Reply:
From what I understand about how yaml parsers work, it doesn't "need" to be below, but It should be moved, yes. reads better that way.

I found this action that should do the trick? https://github.com/marketplace/actions/workflow-conclusion-action 

Assuming @awesomekling  would be ok adding a third party action.

Reply:
ayy looks like that action does the right thing. It even prints out all the good goods in some expandable tabs (jobs, conclusions, conclusion).

Failed run (one job failed): https://github.com/SerenityOS/serenity/runs/1992038457?check_suite_focus=true#step:3:9
Success run (all jobs passed): https://github.com/SerenityOS/serenity/runs/1992047005?check_suite_focus=true#step:3:9